provider/vsphere/environ_policy.go

```diff
@@ -5,11 +5,10 @@ package vsphere
 
 import (
 	"github.com/juju/errors"
-	"github.com/juju/utils/arch"
-
 	"github.com/juju/juju/core/constraints"
 	"github.com/juju/juju/environs"
 	"github.com/juju/juju/environs/context"
+	"github.com/juju/utils/arch"
```

Comment:
This should be back where it was removed from. Only juju/juju gets grouped together :+1: